### PR TITLE
Feature/add post delete vdepatla

### PR DESCRIPTION
**Pull request by vdepatla**
# Description

Added POST and DELETE endpoints

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested by calling the api directly.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: examples/customHttpClient/main.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 func main() {
 
-	tmdbClient, err := tmdb.Init(os.Getenv("APIKey"))
+	tmdbClient, err := tmdb.Init(os.Getenv("APIKey"), os.Getenv("SessionID"))
```
Comment (cyruzin) [inline examples/customHttpClient/main.go]: Hi, this new parameter here will break the current implementation when someone update the lib. Perharps a new function is a better alternatine, like "setSessionID()". What do you think?

Comment (vdepatla) [reply examples/customHttpClient/main.go]: yeah .. since POST and DELETE needs sessionid .. how about these methods take-in a sessionid ?

Comment (cyruzin) [reply examples/customHttpClient/main.go]:
```
(c *Client) SetSessionID(sid string) error { 
  if sid == "" { 
    return errors.New("The SessionID is empty")
  }
  c.sessionID = sid
}

import "github.com/cyruzin/golang-tmdb"

tmdbClient, err := tmdb.Init(os.GetEnv("TMDB_KEY")

tmdbClient.SetAutoRetry()

// Inside the POST and DELETE
c.SetSessionID("SESSION_ID")
if err != nil {
    fmt.Println(err)
}
```

Comment (vdepatla) [reply examples/customHttpClient/main.go]: c.SetSessionID("SESSION_ID") - in here SESSION_ID is env variable ?

Comment (cyruzin) [reply examples/customHttpClient/main.go]: Yes :) 

```
c.SetSessionID(os.GetEnv("SESSION_ID"))
if err != nil {
    fmt.Println(err)
}
```

Comment (cyruzin) [reply examples/customHttpClient/main.go]: I was thinking, this way, we are going to force the users to use ENV variables, though is more secure thay way. You could move this function near the initialization, like this:

tmdbClient, err := tmdb.Init(os.Getenv("APIKey"))
if err != nil {
 log.Println(err)
}

tmdbClient.SetSessionID(os.GetEnv("SESSION_ID"))

Comment (vdepatla) [reply examples/customHttpClient/main.go]: will send new PR

**Pull request closed without merging**